it/support/EmailStub.scala

```diff
@@ -76,6 +76,10 @@ object EmailStub extends MockitoSugar with Matchers {
     Json.parse(emailSendRequest)
   }
 
+  def verifyNoEmailSent = {
+    WireMock.findAll(postRequestedFor(urlEqualTo("/hmrc/email"))).asScala.size shouldBe 0
```

Comment:
```suggestion
    WireMock.findAll(postRequestedFor(urlEqualTo("/hmrc/email"))) should have size 0
```
should work (or maybe `shouldBe empty`) and would avoid the awkward "1 was not equal to 0" type of error

Reply:
👍